mule-packager/src/main/java/org/mule/tools/api/packager/filter/DependenciesFilter.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.tools.api.packager.filter;
+
+import org.mule.tools.api.packager.filter.predicate.*;
+import org.mule.tools.api.packager.packaging.Exclusion;
+import org.mule.tools.api.packager.packaging.Inclusion;
+import org.mule.tools.api.util.Artifact;
+
+import java.util.*;
+
+import static com.google.common.collect.Sets.newHashSet;
+import static java.util.Collections.unmodifiableSet;
+
+/**
+ * Filter to mule project dependencies.
+ */
+public class DependenciesFilter {
```

Comment:
Could we please add a test for this class, it looks important we don't want it failing randomly 💃

Reply:
Done!